pages.ml/common/git.md

```diff
@@ -0,0 +1,30 @@
+# git
+
+> പ്രോഗ്രാമുകളുടെ പല പതിപ്പുകൾ പലയിടങ്ങളിലായി സൂക്ഷിക്കുവാനും നിയന്ത്രിക്കുവാനും ഉള്ള സംവിധാനം
+> `commit`, `add`, `branch`, `checkout`, `push` പോലുള്ള ഉപകമാന്റുകൾക്ക് പ്രത്യേക ഡോക്യൂമെന്റഷൻ ഉണ്ട്.
+> അതിനായി `tldr git {{ഉപകമാൻഡ്}}` ഉപയോഗിക്കുക.
+> കൂടുതൽ വിവരങ്ങൾ: <https://git-scm.com/>.
+
+- നിങ്ങൾ ഉപയോഗിക്കുന്ന ഗിറ്റിന്റെ വേർഷൻ കാണുവാൻ:
+
+`git --version`
+
+- സഹായ നിർദേശങ്ങൾ കാണിക്കുക:
+
+`git --help`
+
+- `clone`, `add`, `push`, `log` പോലുള്ള ഉപകമാൻഡുകളുടെ സഹായ നിർദേശങ്ങൾ കാണുവാൻ:
+
+`git help {{ഉപകമാൻഡ്}}`
+
+- ഗിറ്റ് ഉപകമാന്റുകൾ എക്സിക്യൂട്ട് ചെയ്യുവാൻ:
+
+`git {{ഉപകമാൻഡ്}}`
+
+- ഒരു ഇഷ്‌ടാനുസൃത റിപ്പോസിറ്ററി റൂട്ട് പാത്തിൽ ഒരു git ഉപകമാൻഡ് എക്‌സിക്യൂട്ട് ചെയ്യുവാൻ:
+
+`git -C {{പാത/റിപ്പോസിറ്ററി}} {{ഉപകമാൻഡ്}}`
+
+- കോൺഫിഗറേഷൻ ഉപയോഗിച്ച് ഒരു Git ഉപകമാൻഡ് എക്സിക്യൂട്ട് ചെയ്യുക:
+
+`git -c '{{config.key}}={{value}}' {{ഉപകമാൻഡ്}}`
```

Comment:
Can you translate the `value` command argument too?

Reply:
Would this be better @akash1729 ?

```suggestion
`git -c '{{config.key}}={{മൂല്യം}}' {{ഉപകമാൻഡ്}}`
```

Reply:
yeah മൂല്യം would be better